### PR TITLE
FIX rawDataPacket not being emitted

### LINKED ISSUE #47: `rawDataPacket` not being emitted
There is a bug where `rawDataPacket` is not being emitted.

Two things needs to happen:

1) This bug needs to be patched
2) An automated test needs to be put in place for this!

### PR DESCRIPTION
**Pull request by pushtheworldllc**
Fixes #47

### CODE REVIEW

File: openBCIBoard.js [hunk 1]
```diff
@@ -1080,6 +1080,7 @@ function OpenBCIFactory() {
             while (readingPosition <= bytesToRead - k.OBCIPacketSize) {
                 if (data[readingPosition] === k.OBCIByteStart) {
                     var rawPacket = data.slice(readingPosition, readingPosition + k.OBCIPacketSize);
+                    this.emit('rawDataPacket',rawPacket);
```
Comment (teonbrooks) [inline openBCIBoard.js]: what interprets the `rawDataPacket`? I thought it was only the `sample` that gets emitted

Comment (pushtheworldllc) [inline openBCIBoard.js]: well you had originally asked for this with the EDF thing I believe, but this allows the user to interpret the data packet. It bypasses sample, so nothing interprets it.

Comment (teonbrooks) [inline openBCIBoard.js]: ah, I didn't realize it was even implemented.

**Comment by teonbrooks** [comment]
thanks @pushtheworldllc

**Pull request merged**